tests/FilesystemRepositoryCopyTest.php

```diff
@@ -178,4 +179,20 @@ public function testRemoveFailsIfLanguageNotGlob()
     {
         $this->writeRepo->remove('/*', 'foobar');
     }
+
+    /**
+     * @expectedException \Puli\Repository\Api\UnsupportedResourceException
+     */
+    public function testFileLink()
+    {
+        $this->writeRepo->add('/webmozart/link', new LinkResource('/webmozart/puli/file'));
+    }
+
+    /**
+     * @expectedException \Puli\Repository\Api\UnsupportedResourceException
+     */
+    public function testDirectoryLink()
+    {
+        $this->writeRepo->add('/webmozart/link', new LinkResource('/webmozart/puli/file'));
```

Comment:
I use symlink to implement LinkResource in FilesystemRepository, so if symlinks are disable, LinkResource is not supported.